### PR TITLE
Reporting Pages Design refresh

### PR DESCRIPTION
**Pull request by padams**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Tested the changes
- [x] Build (`/path/to/php cli.php cmd=build`) was run locally and any changes were pushed

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [x] Other (please describe): 

 General CSS tweaks

## What is the current behavior?

Issue Number: #554 among others.

## What is the new behavior?

- refactored pie charts to use labels instead of a legend.
- made report navigation into a proper accordion/collapsable stack 
- added font awesome icons to the project and top level report groups.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

Screen Shot 2020-04-20 at 11 19 35 AM

### CODE REVIEW

**Comment by Maaiins** [comment]
Oh wow, looks very nice

File: modules/base/templates/report.tpl [hunk 1]
```diff
@@ -25,9 +38,9 @@ jQuery(document).ready(function(){
         <TR>
             <?php if ( ! $this->get( 'hideReportingNavigation' ) ):?>
             <TD valign="top" class="owa_reportLeftNavColumn">
-                <div class="reportSectionContainer">
+                <div class="">
```
Comment (Maaiins) [inline modules/base/templates/report.tpl:41]: There´s a missing class. Should it be like this? I would recommend to remove it when the class is empty

File: modules/base/templates/report_dashboard.tpl [hunk 2]
```diff
@@ -51,12 +47,22 @@
...
             </div>
             <?php //} ?>
-
-            <div class="owa_reportSectionContent">
+			<table width="100%">
+				<TR>
+					<TD>
+			 <span class="owa_reportSectionContent">
+                <div class="owa_reportSectionHeader">Visitor Types</div>
+                <div id="visitor-types" style="width:250px;margin-top:-10px;"></div>
```
Comment (Maaiins) [inline modules/base/templates/report_dashboard.tpl:55]: Shouldn´t that moved to css files?

File: modules/base/templates/report_dashboard.tpl [hunk 3]
```diff
@@ -51,12 +47,22 @@
...
+            </span>
+					</TD>
+					<TD>
+            <span class="owa_reportSectionContent" >
                 <div class="owa_reportSectionHeader">Traffic Sources</div>
                 <div id="visitor-mediums" style="width:250px;margin-top:-10px;"></div>
```
Comment (Maaiins) [inline modules/base/templates/report_dashboard.tpl:61]: Shouldn´t that moved to css files?

Comment (Maaiins) [inline modules/base/templates/report_dashboard.tpl:61]: Shouldn´t that moved to css files?

File: modules/base/templates/report.tpl (hunk 1 above)
Comment (padams) [reply modules/base/templates/report.tpl:41]: agreed.

File: modules/base/templates/report_dashboard.tpl (hunk 3 above)
Comment (padams) [reply modules/base/templates/report_dashboard.tpl:61]: yes.

**Comment by padams** [comment]
You have a  conflict here as well. looks like the timestamp in owa-tracker. Maybe do a rebuild as well so the timestamp is current.

**Comment by Maaiins** [comment]
> You have a conflict here as well. looks like the timestamp in owa-tracker. Maybe do a rebuild as well so the timestamp is current.

That´s your pull request ;-) Just rebuild owa tracker

**Pull request merged**